tests/TestCase/Shell/CompletionShellTest.php

```diff
@@ -116,7 +116,7 @@ public function testCommands()
         $output = $this->out->output;
 
         $expected = 'TestPlugin.example TestPlugin.sample TestPluginTwo.example unique welcome ' .
-            "cache i18n orm_cache plugin routes server i18m sample testing_dispatch\n";
+            "cache i18n orm_cache plugin routes server version i18m sample testing_dispatch\n";
```

Comment:
Yes its an intentional typo in the classname. Not sure from whence it came, but its definitely there.